packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -425,29 +418,25 @@ export async function activate(context: vscode.ExtensionContext) {
     decorators.showDemoMode();
   }
 
-  // Refresh SObject definitions if there aren't any faux classes
-  if (sfdxCoreSettings.getEnableSObjectRefreshOnStartup()) {
-    initSObjectDefinitions(getRootWorkspacePath()).catch(e =>
-      telemetryService.sendErrorEvent(e.message, e.stack)
-    );
-  }
-
   const api: any = {
-    ProgressNotification,
+    channelService,
     CompositeParametersGatherer,
     EmptyParametersGatherer,
+    getDefaultUsernameOrAlias,
+    getRootWorkspacePath,
```

Comment:
I don't think we should expose `getRootWorkspacePath` to other extensions. It is a nice wrapper to the vscode workspace api but it does not add anything to justify other extensions using it.